lib/index.js

```diff
@@ -22,11 +22,15 @@ webFrame.setZoomLevelLimits(1, 1);
 
 const store_ = createStore(
   rootReducer,
-  applyMiddleware(
-    thunk,
-    plugins.middleware,
-    thunk,
-    effects
+  compose(
+    applyMiddleware(
+      thunk,
+      plugins.middleware,
+      thunk,
+      effects
+    ),
+    // Redux DevTools extension integration
+    window.devToolsExtension ? window.devToolsExtension() : f => f
```

Comment:
Just curious what `window.devToolsExtension ? window.devToolsExtension() : f => f` does?
Could it be simplified to: `window.devToolsExtension ? window.devToolsExtension() : false`? Since I don't see why there should be an arrow function there?